wolfssl/test.h

```diff
@@ -3574,6 +3574,15 @@ static WC_INLINE const char* mymktemp(char *tempfn, int len, int num)
 
     #include <wolfssl/wolfcrypt/chacha20_poly1305.h>
 
+#ifdef WOLFSSL_SNIFFER
+    static const byte sessTicketKey[CHACHA20_POLY1305_AEAD_KEYSIZE] = {
```

Comment:
FYI: This is the static testing key for session tickets with the sniffer. Ideally we can resolve the ticketNonce and eliminate this.